common/modal.js

```diff
@@ -76,6 +76,7 @@
 
             params.displayname = refTable.name+'_'+tuple.uniqueId,
             params.reference = reference;
+            if (reference.citation) params.title = "and Cite";
```

Comment:
The `params` might already have `title` in it (in viewer app, I'm changing the title of share popup for annotations. You can find an example [here](https://dev.rebuildingakidney.org/chaise/viewer/#2/Gene_Expression:Image/id=14-47YP)) So you shouldn't blindly just change it. Something like this:

```javascript
if (reference.citation) {
  var title = "and Cite";
  if (params.title) {
    title += " " + params..title;
  }
  params.title = title;
}
```